src/ui/molecules/payment-button.svelte

```diff
@@ -3,13 +3,41 @@
   import Button from "../atoms/button.svelte";
   import Localized from "../localization/localized.svelte";
   import { LocalizationKeys } from "../localization/supportedLanguages";
+  import { type PriceBreakdown } from "../ui-types";
+  import { type Writable } from "svelte/store";
+  import { Translator } from "../localization/translator";
+  import { translatorContextKey } from "../../ui/localization/constants";
+  import { getContext } from "svelte";
 
-  export let disabled: boolean;
-  export let subscriptionOption: SubscriptionOption | null;
+  type Props = {
+    disabled: boolean;
+    subscriptionOption: SubscriptionOption | null;
+    priceBreakdown?: PriceBreakdown;
+    wallet?: string;
```

Comment:
I anticipate that we will need to do the same for `cards`
```suggestion
    payment_method?: string;
```